test/kubemark/start-kubemark.sh

```diff
@@ -80,6 +84,10 @@ done
 rm kubemark
 cd $CURR_DIR
 
+##################################################################################
```

Comment:
nit: I know it's pretty common in different project, but in Kubernetes we are trying to avoid things like long lines with "#" "or" = or "-" but we are trying to avoid these here.